content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -82,22 +93,22 @@ cluster for mesh expansion, run the following commands on a machine with cluster
     $ echo -e "ISTIO_CP_AUTH=MUTUAL_TLS\nISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR\n" > cluster.env
     {{< /text >}}
 
-1.  Check the contents of the generated `cluster.env` file. It should be similar to the following example:
+1. Check the contents of the generated `cluster.env` file. It should be similar to the following example:
 
     {{< text bash >}}
     $ cat cluster.env
     ISTIO_CP_AUTH=MUTUAL_TLS
     ISTIO_SERVICE_CIDR=10.55.240.0/20
     {{< /text >}}
 
-1.  (Optional)  If the VM only calls services in the mesh, you can skip this step. Otherwise, add the ports the VM exposes
+1. (Optional)  If the VM only calls services in the mesh, you can skip this step. Otherwise, add the ports the VM exposes
     to the `cluster.env` file with the following command. You can change the ports later if necessary.
 
     {{< text bash >}}
     $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> cluster.env
```

Comment:
It is worthwhile to mention that 3306/8080 are just examples and people should change this accordingly in their scenario.

Reply:
We have this above
> Otherwise, add the ports the VM exposes  to the `cluster.env` file with the following command.

Do you think we should be more specific and clear?